Problem3/Problem3.js

```diff
@@ -0,0 +1,7 @@
+function checkForDuplicates(array){
+    for (var i = 0; i < array.length; i++){
+        if (array.indexOf(array[i]) !== array.lastIndexOf(array[i]))
```

Comment:
`indexOf` and `lastIndexOf` loop over array items and return on index.

Cu alte cuvinte nu ai respectat cerinta din problema sa nu folosesti 2 nesting loops.